CMakeLists.txt

```diff
@@ -108,7 +114,7 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR}/include)
 
 function(torch_mlir_target_includes target)
   set(_dirs
-    $<BUILD_INTERFACE:${MLIR_INCLUDE_DIR}>
+    $<BUILD_INTERFACE:${MLIR_INCLUDE_DIRS}>
```

Comment:
I was surprised to see this change. Does it suggest we should also remove the `MLIR_GENERATED_INCLUDE_DIR` from this file too?

Reply:
Yes you're right!